backend/tests/logger_test.py

```diff
@@ -0,0 +1,116 @@
+import unittest
+from unittest import mock
+from mock import MagicMock
+from mock import patch
+import sys
+import os
+
+sys.path.append("../logger")
+from file_handler import FileHandler
+
+from logger import Logger, Level
+
+
+class LoggerTest(unittest.TestCase):
+    def setup(self, level):
+        self.logger = Logger("Logger_Test", level)
+        self.logger.level = level
+        self.writer = MagicMock()
+
+    def test_singleton_design_pattern(self):
+        logger1 = Logger()
+        logger2 = Logger()
+        assert logger1 is logger2
+
+    @patch("file_handler.FileHandler.get_write_handle")
+    def test_debug_level_write_debug(self, _handle):
```

Comment:
Please adhere to the mock convention here as well please 😄 (and in other places in the file)